.github/renovate.json

```diff
@@ -3,7 +3,7 @@
     "config:base"
   ],
   "dependencyDashboard": true,
-  "dependencyDashboardApproval": false,
+  "dependencyDashboardApproval": true,
```

Comment:
```suggestion
  "dependencyDashboardApproval": false,
```

Reply:
This change was undone in the last commit ..should it apply the suggestion again?